ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/index.tsx

```diff
@@ -265,8 +304,9 @@ const RunComponent = () => {
                   className={`h-4 w-4 ${getStatusFontColor(ortRun.jobs.analyzer?.status)}`}
                 />
               )}
-              value={ortRun.jobs.analyzer ? '-' : 'N/A'}
-              className='hover:bg-muted/50'
+              value={ortRun.jobs.analyzer ? packagesTotal : 'N/A'}
+              description={`from ${detectedPackageManagers.length} package managers (${detectedPackageManagers.join(`,`)})`}
```

Comment:
IMO this would look better with a badge for each package manager instead of a sentence.

Reply:
I think a sentence is fine for now. For monorepos with several package managers badges would probably take up too much screen space.